ouroboros-network/src/Ouroboros/Network/TipSample/Server.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE DataKinds           #-}
+{-# LANGUAGE KindSignatures      #-}
+{-# LANGUAGE NamedFieldPuns      #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Ouroboros.Network.TipSample.Server where
+
+import           Control.Monad.Class.MonadSTM
+
+import           Network.TypedProtocol.Pipelined (Nat (Succ, Zero), N (..))
+
+import           Cardano.Slotting.Slot (SlotNo)
+
+import           Ouroboros.Network.Block (Tip (..))
+import           Ouroboros.Network.Protocol.TipSample.Server
+
+
+-- | 'tip-sample' server application.
+--
+tipServer :: forall header m.
+             MonadSTM m
+          => STM m (Tip header)
+          -- ^ 'ChainDB' 'getCurrentSlot' method.
```

Comment:
Is the comment or the variable name correct?
That is is this a `getCurrentSlot` method or a `getCurrentTip` method?

Reply:
it should be `getCurrentTip`.

Reply:
```suggestion
          -- ^ 'ChainDB' 'getCurrentTip' method.
```